README.md

```diff
@@ -1,24 +1,24 @@
 
-# Amazon EKS SSP Quick Start
+# Amazon EKS Blueprints Quick Start
 
```

Comment:
Amazon EKS Blueprints for CDK...don't think you need quickstart?

Reply:
addressed